src/meta-srv/src/bootstrap.rs

```diff
@@ -275,8 +276,15 @@ async fn create_postgres_client(opts: &MetasrvOptions) -> Result<tokio_postgres:
     let postgres_url = opts.store_addrs.first().context(InvalidArgumentsSnafu {
         err_msg: "empty store addrs",
     })?;
-    let (client, _) = tokio_postgres::connect(postgres_url, NoTls)
+    let (client, connection) = tokio_postgres::connect(postgres_url, NoTls)
         .await
         .context(error::ConnectPostgresSnafu)?;
+
+    tokio::spawn(async move {
+        connection
+            .await
+            .context(error::PostgresExecutionSnafu)
+            .unwrap()
+    });
```

Comment:
It doesn't seem necessary?

Reply:
I actually follows the document of `tokio_postgres` [here](https://docs.rs/tokio-postgres/latest/tokio_postgres/struct.Connection.html):
> This is one half of what is returned when a new connection is established. It performs the actual IO with the server, and should generally be spawned off onto an executor to run in the background.

Not quite sure what will happen if not spawn a test with it.